scripts/start.mts

```diff
@@ -0,0 +1,178 @@
+const helpIndex = args.findIndex((arg) => arg === '-h' || arg === '--help');
+
+const isOurHelp =
+  helpIndex !== -1 && (firstTargetIndex === -1 || helpIndex < firstTargetIndex);
+
+if (isOurHelp) {
+  console.log(
+    await fs.readFile(path.join(import.meta.dirname, 'start.md'), 'utf8')
+  );
+  process.exit(0);
+}
+
+// Parse positional arguments and group them by target
+const targets = {
+  desktop: { enabled: false, args: [] as string[] },
+  sandbox: { enabled: false, args: [] as string[] },
+  sync: { enabled: false, args: [] as string[] },
+};
+
+let currentTarget: keyof typeof targets | null = null;
+
+for (const arg of args) {
+  if (validTargets.includes(arg as keyof typeof targets)) {
+    // Switch to new target
+    currentTarget = arg as keyof typeof targets;
+    targets[currentTarget].enabled = true;
+  } else if (currentTarget) {
+    // Add argument to current target's args (including help flags destined for targets)
+    targets[currentTarget].args.push(arg);
+  }
+  // If arg is not a valid target and no current target, ignore it
+}
+
+// If no targets specified, default to desktop
+if (
+  !targets.desktop.enabled &&
+  !targets.sandbox.enabled &&
+  !targets.sync.enabled
+) {
+  targets.desktop.enabled = true;
+}
+
+const subProcesses: child_process.ChildProcess[] = [];
+async function cleanup(signal: NodeJS.Signals) {
+  for (const p of subProcesses) p.kill(signal);
+  console.log('\nstart    | requested termination.');
+  await timers.setTimeout(5000);
+  const stillRunning = subProcesses.filter((p) => p.exitCode === null);
+  for (const p of stillRunning) p.kill('SIGKILL');
+  console.log('\nstart    | done.');
+  process.exit(0);
+}
+
+process.on('SIGINT', cleanup).on('SIGTERM', cleanup);
+
+// Helper function to create a transform stream that prefixes lines
+function createPrefixTransform(prefix: string) {
+  let buffer = '';
+  return new Transform({
+    transform(chunk, _, callback) {
+      buffer += chunk.toString();
+      const lines = buffer.split('\n');
+      // Keep the last line in buffer (might be incomplete)
+      buffer = lines.pop() || '';
+      // Process complete lines
+      for (const line of lines) {
+        this.push(`${prefix} | ${line}\n`);
+      }
+      callback();
+    },
+
+    flush(callback) {
+      // Process any remaining data in buffer
+      if (buffer) {
+        this.push(`${prefix} | ${buffer}\n`);
+      }
+      callback();
+    },
+  });
+}
+
+// Helper function to spawn a target process
+function spawnTarget(
+  command: string,
+  workspace: string,
+  args: string[],
+  targetName: string
+) {
+  // Only set color-forcing env vars if user hasn't set color preferences
+  const colorEnv: Record<string, string> = {};
+  if (
+    !process.env.NO_COLOR &&
+    !process.env.FORCE_COLOR &&
+    !process.env.CLICOLOR_FORCE
+  ) {
+    colorEnv.FORCE_COLOR = '1';
+    colorEnv.CLICOLOR_FORCE = '1';
+  }
```

Comment:
Respect user set prefs first, then force the subcommands to use color because their not writing to a tty anymore. But we want color ofc! 🌈